src/go/k8s/Makefile

```diff
@@ -1,6 +1,7 @@
 
 # Image URL to use all building/pushing image targets
 IMG ?= vectorized/redpanda-operator:latest
+CONFIGURATOR_IMG ?= gcr.io/vectorized/configurator:latest
```

Comment:
I think we'll need this in dockerhub as well, right?

Reply:
Yes, I will change this to docker hub now.